src/ripple/json/impl/json_reader.cpp

```diff
@@ -925,7 +924,15 @@ Reader::getLocationLineAndColumn(Location location) const
     int line, column;
     getLocationLineAndColumn(location, line, column);
     char buffer[18 + 16 + 16 + 1];
-    sprintf(buffer, "Line %d, Column %d", line, column);
+    int generatedStrLen =
+        snprintf(buffer, 51, "Line %d, Column %d", line, column);
```

Comment:
here is the documentation for the `snprintf` function: https://cplusplus.com/reference/cstdio/snprintf/

I'm basing the second parameter off of the previous line. But I don't understand why `buffer` needs to have a length of 50. The contents of `buffer` seem to require much lesser space.